arangod/Aql/QueryContext.h

```diff
@@ -36,6 +36,12 @@
 #include "VocBase/voc-types.h"
 #include <velocypack/Builder.h>
 
+#ifdef __APPLE__
+#include <experimental/memory_resource>
+#else
+#include <memory_resource>
+#endif
```

Comment:
Could you write this and typedef single time in the single header?
Also it's not correct at least for libc++ and linux